catalog/styles/header.css

```diff
@@ -0,0 +1,332 @@
+  align-items: center;
+  position: relative;
+  left: 83px;
+  bottom: 24px;
+}
+
+.main-screen__product-pic {
+  margin-top: 84px;
+}
+
+#header__nav--list {
+  display: flex;
+  justify-content: space-between;
+  width: 477px;
+  max-width: 100%;
+}
+
+#header__nav__toggle {
+  visibility: hidden;
+}
+
+#header__nav--list {
+  visibility: visible;
+}
+
+.main-screen__product-desktop {
+  margin: 22px 0 60px 0;
+}
+
+.main-screen__product-tablet {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  white-space: nowrap;
+  clip-path: inset(100%);
+  clip: rect(0 0 0 0);
+  overflow: hidden;
+}
+
+.header__logo--desktop {
+  width: 202px;
+  height: 70px;
+}
+
+.header__logo--mobile {
+  width: 38px;
+  height: 38px;
+}
+
+.header__header__logo {
+  visibility: hidden;
+}
+
+/* БУРГЕР */
+.header__nav {
+  position: relative;
+}
+
+#header__nav__toggle {
+  z-index: 100;
+  -webkit-user-select: none;
+  user-select: none;
+}
+
+#header__nav__toggle a:hover {
+  color: #68b738;
+}
+
+#header__nav__toggle input {
+  display: block;
+  width: 40px;
+  height: 32px;
+  position: absolute;
+  top: -10px;
+  left: -12px;
+  cursor: pointer;
+  opacity: 0;
+  z-index: 300;
+}
+
+#header__nav__toggle span {
+  display: block;
+  width: 28px;
+  height: 2px;
+  margin-bottom: 6px;
+  position: relative;
+  background: #222222;
+  border-radius: 3px;
+  z-index: 300;
+  transform-origin: 4px 0px;
+  transition: transform 0.5s cubic-bezier(0.77, 0.2, 0.05, 1),
+    background 0.5s cubic-bezier(0.77, 0.2, 0.05, 1), opacity 0.55s ease;
+}
+
+#header__nav__toggle span:first-child {
+  transform-origin: 0% 0%;
+}
+
+#header__nav__toggle span:nth-last-child(2) {
+  transform-origin: 0% 100%;
+}
+
+#header__nav__toggle input:checked ~ span {
+  opacity: 1;
+  transform: rotate(45deg) translate(-2px, -1px);
+  background: #232323;
+}
+
+#header__nav__toggle input:checked ~ span:nth-last-child(3) {
+  opacity: 0;
+  transform: rotate(0deg) scale(0.2, 0.2);
+}
+
+#header__nav__toggle input:checked ~ span:nth-last-child(2) {
+  transform: rotate(-45deg) translate(0, -1px);
+}
+
+#header__nav__toggle input:checked ~ ul {
+  transform: none;
+}
+
+@media (max-width: 1024px) {
+  .main-screen {
+    background-image: inherit;
+  }
+
+  .main-screen__text-wrapper {
+    margin-top: 67px;
+    margin-bottom: 0;
+    position: relative;
+    z-index: 90;
+  }
+
+  .main-screen__product-desktop {
+    margin: 0;
+    position: relative;
+    top: -90px;
+    left: 80px;
+  }
+
+  .header__link {
+    font-size: 20px;
+    color: #111111;
+    white-space: nowrap;
+  }
+
+  .header__link:hover {
+    opacity: 0.6;
+  }
+
+  .header__link:active {
+    opacity: 0.3;
+  }
+
+  .header__link--active::after {
+    background-color: #68b738;
+  }
+}
+
+@media (max-width: 768px) {
+  .header {
+    padding-top: 4px;
+  }
+
+  .header__logo-link {
+    padding-top: 20px;
+  }
+
+  #header__nav--list {
```

Comment:
Please check IDs in all files and replace them with CSS classes, we don't use IDs for styling